### PR TITLE
feat(home): new home screen

### PR DESCRIPTION
**Pull request by jh2oman**
### Description

[Figma
](https://www.figma.com/design/PzsUxBv03Kg6GgzsSq5R1D/Mobile-Stack-x-Mento?node-id=2143-22293&node-type=frame&m=dev)

[Linear](https://linear.app/valora/issue/ACT-1433/mento-app-new-home)

New Home screen for mento

https://github.com/user-attachments/assets/dc50ff40-0171-439e-8836-1a81f1043570

### Test plan

Added unit tests

### Related issues

https://linear.app/valora/issue/ACT-1433/mento-app-new-home

### CODE REVIEW

File: src/analytics/docs.ts [hunk 1]
```diff
@@ -612,6 +613,13 @@ export const eventDocs: Record<AnalyticsEventType, string> = {
   [EarnEvents.earn_pool_info_tap_safety_details]: `When the user taps the view more/less details on the safety card on the pool info screen`,
   [EarnEvents.earn_select_withdraw_type]: `When a user selects a withdrawal type on the withdraw bottom sheet`,
 
+  [TabHomeEvents.add_ckes]: 'Mento app',
```
Comment (satish-ravi) [inline src/analytics/docs.ts]: is this just a placeholder value? I think its obvious from where this code lives that these are all of mento. Would prefer to have a better description here. It wasn't obvious to me that `add_ckes_from_cash_in` and `add_ckes_from_swap` are from the bottom sheet for example (also, should they be a single event, with a property)?

File: src/home/TabHome.tsx [hunk 2]
```diff
@@ -74,13 +92,222 @@ function TabHome(_props: Props) {
...
 const styles = StyleSheet.create({
   container: {
     flex: 1,
     position: 'relative',
+    paddingHorizontal: Spacing.Regular16,
+    gap: Spacing.Regular16,
+  },
+  flatCard: {
+    backgroundColor: 'white',
+    padding: 16,
```
Comment (satish-ravi) [inline src/home/TabHome.tsx]: use Spacing constant

File: src/home/TabHome.tsx [hunk 3]
```diff
@@ -74,13 +92,222 @@ function TabHome(_props: Props) {
...
+    borderColor: Colors.black,
+    borderWidth: 1,
+  },
+  column: {
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    gap: Spacing.Smallest8,
+  },
+  labelSemiBoldMedium: {
```
Comment (satish-ravi) [inline src/home/TabHome.tsx]: Here and below, needs to specify color explicitly, the default black isn't the same as Colors.black.
Also, is it possible to name this semantically (e.g., `actionText` or something similar). Not a fan of reusing the typescale name here

File: src/icons/Send.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import * as React from 'react'
```
Comment (satish-ravi) [inline src/icons/Send.tsx]: can you reuse existing icons? I am pretty sure you just need to adjust colors and size and add a stroke

File: src/home/TabHome.tsx [hunk 5]
```diff
@@ -74,13 +92,222 @@ function TabHome(_props: Props) {
...
+function FlatCard({
+  onPress,
+  testID,
+  ...props
+}: {
+  children: React.ReactNode
+  onPress: () => void
+  testID: string
+}) {
+  return <Touchable style={styles.flatCard} testID={testID} onPress={onPress} {...props} />
```
Comment (satish-ravi) [inline src/home/TabHome.tsx]: have you tested this on android? we might need a borderRadius here so the touchable ripple displays correctly

File: src/home/TabHome.tsx [hunk 6]
```diff
@@ -74,13 +92,222 @@ function TabHome(_props: Props) {
...
+  function onPressWithdraw() {
+    AppAnalytics.track(TabHomeEvents.withdraw)
+    navigate(Screens.WithdrawSpend)
+  }
+
+  return (
+    <SafeAreaView testID="WalletHome" style={styles.container} edges={[]}>
+      <FlatCard testID="FlatCard/AddCKES" onPress={onPressAddCKES}>
+        <View style={styles.column}>
+          {!!cKESToken && (
```
Comment (satish-ravi) [inline src/home/TabHome.tsx:146]: I think we're doing this in quite a few places. We probably need some error state if tokens info is unavailable (e.g., if the network is down). Probably fine for now, but maybe required when we release the app. Should we make a placeholder ticket?

File: src/analytics/docs.ts (hunk 1 above)
Comment (jh2oman) [reply src/analytics/docs.ts]: They are completely separate buttons so I think them being separate analytics events makes sense. I can add more description though. I thought it was obvious but you bring up a good point

File: src/icons/Send.tsx (hunk 4 above)
Comment (jh2oman) [reply src/icons/Send.tsx]: I tried, it doesn't look right. There's something different between the path's that changes how the stroke appears

File: src/home/TabHome.tsx (hunk 5 above)
Comment (jh2oman) [reply src/home/TabHome.tsx]: Thanks, you're right. fixed

File: src/home/TabHome.tsx (hunk 6 above)
Comment (jh2oman) [reply src/home/TabHome.tsx:146]: I'm wondering if other error banners will be sufficient to let the user know that something is wrong and thats why the buttons aren't working. We'll have to experiment with trying to load the app when there is a network issue. I can create a ticket

File: src/styles/colors.tsx [hunk 7]
```diff
@@ -24,6 +24,7 @@ enum Colors {
   error = '#EA6042',
   errorDark = '#C93717',
   errorLight = '#FBF2F0',
+  periwinkle = '#869AEF',
```
Comment (satish-ravi) [inline src/styles/colors.tsx]: is this used anywhere?

File: src/icons/Send.tsx (hunk 4 above)
Comment (satish-ravi) [reply src/icons/Send.tsx]: surprising given that the figma references the base icon. Maybe the base icon itself is wrong in the codebase. Anyway, can we group all the home screen specific ones into a folder? So we don't have 2 icons with the same name which could be confusing

**Review by satish-ravi — COMMENTED** [review]
A w/ nits

**Review by satish-ravi — APPROVED** [review]

File: src/styles/colors.tsx (hunk 7 above)
Comment (jh2oman) [reply src/styles/colors.tsx]: whoopps, that was from me trying to make one of the icons work.

**Pull request merged**